### ISSUE TITLE
Relative URLs act like absolute URLs

### ISSUE THREAD

**Issue by trezy** [jmeas/backbone.intercept#15]
I'm utilizing relative URLs in my app to make my life easier with deeply nested URLs. Unfortunately bb.Intercept seems to be mapping my URLs absolutely.
```html
<!-- curent address: http://trezy.com/foo -->
<a href="bar">Bar</a>
<!-- should take me to http://trezy.com/foo/bar, with bb.Intercept enabled takes me to http://trezy.com/bar -->
```

**Comment by jmeas** [comment]
Will-fix. Thanks for the report!

**Comment by jmeas** [comment]
The issue is that I'm looking at the anchor tag's `href` attribute, and not `a.pathname`. The former returns the actual value of the `href` attr (unsurprisingly), whereas the latter is the calculated pathname.

In addition to this, I will need to support a `root` option to put this library in line w. Backbone history's options.

**Comment by jmeas** [comment]
Released as [v0.3.2](https://github.com/jmeas/backbone.intercept/releases/tag/v0.3.2)
